### PR TITLE
PIP support for Jitsi call

### LINKED ISSUE #2418: Can't minimise jitsi call in Android 10?
Apparently in Android 10 nothing is shown in the upper left corner to minimise a Jitsi call

### PR DESCRIPTION
**Pull request by bmarty**
Fixes #2418

Also handle switch to another conference.

### CODE REVIEW

File: vector/src/main/java/im/vector/app/features/call/conference/VectorJitsiActivity.kt [hunk 1]
```diff
@@ -147,13 +181,23 @@ class VectorJitsiActivity : VectorBaseActivity<ActivityJitsiBinding>(), JitsiMee
...
     }
 
-//    override fun onUserLeaveHint() {
-//        super.onUserLeaveHint()
-//        jitsiMeetView?.enterPictureInPicture()
-//    }
+    override fun onUserLeaveHint() {
+        super.onUserLeaveHint()
+        if (packageManager.hasSystemFeature(PackageManager.FEATURE_PICTURE_IN_PICTURE)) {
+            jitsiMeetView?.enterPictureInPicture()
```
Comment (ganfra) [inline vector/src/main/java/im/vector/app/features/call/conference/VectorJitsiActivity.kt:187]: I think compat is already handled by Jitsi

File: vector/src/main/java/im/vector/app/features/call/conference/VectorJitsiActivity.kt [hunk 2]
```diff
@@ -147,13 +181,23 @@ class VectorJitsiActivity : VectorBaseActivity<ActivityJitsiBinding>(), JitsiMee
         super.onDestroy()
     }
 
-//    override fun onUserLeaveHint() {
-//        super.onUserLeaveHint()
-//        jitsiMeetView?.enterPictureInPicture()
-//    }
+    override fun onUserLeaveHint() {
```
Comment (ganfra) [inline vector/src/main/java/im/vector/app/features/call/conference/VectorJitsiActivity.kt:184]: Also, I think this method is only called when pressing home button, what about pressing back?

**Review by ganfra — COMMENTED** [review]
2 small remarks

File: vector/src/main/java/im/vector/app/features/call/conference/VectorJitsiActivity.kt (hunk 1 above)
Comment (bmarty) [reply vector/src/main/java/im/vector/app/features/call/conference/VectorJitsiActivity.kt:187]: Yes, but better safe than sorry :)

File: vector/src/main/java/im/vector/app/features/call/conference/VectorJitsiActivity.kt (hunk 2 above)
Comment (bmarty) [reply vector/src/main/java/im/vector/app/features/call/conference/VectorJitsiActivity.kt:184]: Yes, if you press back, you leave the conference, and I think it is OK, no?

Comment (ganfra) [reply vector/src/main/java/im/vector/app/features/call/conference/VectorJitsiActivity.kt:184]: I don't know, generally I want to leave only when explicitly pressing hang-up

**Review by ganfra — APPROVED** [review]
LGTM, thanks

**Pull request merged**